src/xcode/ENA/ENA/Source/Scenes/Checkin/CheckinCoordinator.swift

```diff
@@ -137,7 +137,7 @@ final class CheckinCoordinator {
 		viewController.present(topBottomContainerViewController, animated: true)
 	}
 
-	private func showQRCodeScanner() {
```

Comment:
move that in the corresponding Internal section?

Reply:
You are right. I moved it.